src/clj/rems/event_notification.clj

```diff
@@ -19,6 +19,9 @@
             "to" (:url target))
   (try
     (let [timeout-ms (* 1000 (get target :timeout default-timeout))
+          body (if (:send-application? target true)
+                 body
+                 (dissoc body :event/application))
```

Comment:
should we do this check in `(notification-body)` already and not fetch the application in the first place? is there a reason we should fetch the application, copy it into outbox table and not use it?

Reply:
Originally I remembered that the logging used it so I delayed it later. 

But as for sending, the notification can go to multiple targets. It could be that one of the targets needs the application, so it would need to be changed to be lazily fetched so as not to fetch it multiple times. If we had a consistent cache then it could be gotten from there, but it could also return a different application for different targets.

Reply:
Ok so I fixed it with a `delay`. It cleaned everything up a bit, I think.